components/playlist/browser/playlist_service.cc

```diff
@@ -177,11 +197,27 @@ bool PlaylistService::RemoveItemFromPlaylist(const PlaylistId& playlist_id,
                           ConvertPlaylistToValue(target_playlist));
   }
 
-  // TODO(sko) Once we can support to share an item between playlists, we should
-  // check if other playlists have this item before deleting this item
-  // permanantly.
-  if (remove_item)
-    DeletePlaylistItemData(*item_id);
+  GetPlaylistItem(
```

Comment:
It would be good to add some comments about this code something like `Try to remove playlist_id from this item's parent list`.